### ISSUE TITLE
Handle invalid floating point colspan values gracefully

### ISSUE THREAD

**Issue by jerridan** [matthewwithanm/python-markdownify#123]
I ran into an issue where I was trying to convert html that had invalid floating point `colspan` values. This results in an error being thrown when `markdownify` attempts to convert it to an integer.

Ideally, I could fix the offending html, but I don't own that html. Rather than throwing an error, I think `markdownify` should just round to the nearest integer.

**Example html:**
```
<tr>
  <td colspan="0.75">my colspan</td>
</tr>
```

**Expected behaviour:**
returns `\n| my colspan\n\n`

**Actual behaviour:**
Throws `ValueError: invalid literal for int() with base 10: '0.75'`

**Comment by LeMoussel** [comment]
Same error with this Wikipedia page [Ford](https://fr.wikipedia.org/wiki/Ford)
```
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/dev/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 39, in <module>
    cli.main()
  File "/home/dev/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 430, in main
    run()
  File "/home/dev/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 284, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/home/dev/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 321, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/home/dev/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 135, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/home/dev/.vscode-server/extensions/ms-python.debugpy-2024.4.0-linux-x64/bundled/libs/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "html_to_markdown.py", line 24, in <module>
    markdown_content  = MarkdownConverter().convert_soup(soup)
  File "/home/dev/.local/lib/python3.10/site-packages/markdownify/__init__.py", line 100, in convert_soup
    return self.process_tag(soup, convert_as_inline=False, children_only=True)
  File "/home/dev/.local/lib/python3.10/site-packages/markdownify/__init__.py", line 143, in process_tag
    text += self.process_tag(el, convert_children_as_inline)
  File "/home/dev/.local/lib/python3.10/site-packages/markdownify/__init__.py", line 143, in process_tag
    text += self.process_tag(el, convert_children_as_inline)
  File "/home/dev/.local/lib/python3.10/site-packages/markdownify/__init__.py", line 143, in process_tag
    text += self.process_tag(el, convert_children_as_inline)
  [Previous line repeated 16 more times]
  File "/home/dev/.local/lib/python3.10/site-packages/markdownify/__init__.py", line 148, in process_tag
    text = convert_fn(node, text, convert_as_inline)
  File "/home/dev/.local/lib/python3.10/site-packages/markdownify/__init__.py", line 381, in convert_td
    colspan = int(el['colspan'])
ValueError: invalid literal for int() with base 10: ''
```

**Comment by stutteringp0et** [comment]
@LeMoussel 
> ```
> ValueError: invalid literal for int() with base 10: ''
> ```
I ran across this exact error, and it turns out that an empty colspan or rowspan value causes the error, for example:
```<td colspan>```
In my case, I resolved it with some regex replacements on colspan and rowspan not followed by an "=" before processing markdown.
```html_content = re.sub(r'(colspan|rowspan)(?!=)', '', html_content)```
I hope that helps.

**Comment by AlexVonB** [comment]
Thanks for reporting this! Dupicate of #126, closed with 2ec33384de85d0906b4b40a59f1a3650846150cb
